spring/src/main/java/org/axonframework/spring/serialization/avro/ClasspathAvroSchemaLoader.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2010-2024. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.spring.serialization.avro;
+
+import org.apache.avro.Schema;
+
+import java.util.List;
+
+/**
+ * Classpath Avro schema loader.
+ */
+public interface ClasspathAvroSchemaLoader {
```

Comment:
Ah, this is your own custom loader. Then I think my comment on the auto-config class is invalid.